saleor/tests/e2e/checkout/test_checkout_create_order_with_no_payment.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+
+from ..channel.utils import create_channel
+from ..product.utils import (
+    create_category,
+    create_product,
+    create_product_channel_listing,
+    create_product_type,
+    create_product_variant,
+    create_product_variant_channel_listing,
+)
+from ..shipping_zone.utils import (
+    create_shipping_method,
+    create_shipping_method_channel_listing,
+    create_shipping_zone,
+)
+from ..utils import assign_permissions
+from ..warehouse.utils import create_warehouse
+from .utils import (
+    checkout_create,
+    checkout_delivery_method_update,
+    raw_checkout_complete,
+)
+
+
+def prepare_product(
+    e2e_staff_api_client,
+    permission_manage_products,
+    permission_manage_channels,
+    permission_manage_shipping,
+    permission_manage_product_types_and_attributes,
+    channel_slug,
+):
+    permissions = [
+        permission_manage_products,
+        permission_manage_channels,
+        permission_manage_shipping,
+        permission_manage_product_types_and_attributes,
+    ]
+    assign_permissions(e2e_staff_api_client, permissions)
+
+    warehouse_data = create_warehouse(e2e_staff_api_client)
+    warehouse_id = warehouse_data["id"]
+
+    warehouse_ids = [warehouse_id]
+    channel_data = create_channel(
+        e2e_staff_api_client,
+        slug=channel_slug,
+        warehouse_ids=warehouse_ids,
+        allow_unpaid_orders=True,
+        automatically_fulfill_non_shippable_giftcard=True,
```

Comment:
I think made a mistake in the testmo in this test `automaticallyConfirmAllNewOrders` should be set here to true not `automaticallyFulfillNonShippableGiftCard` Can you update the code and testmo?

Reply:
https://github.com/saleor/saleor/pull/13576/commits/554d4bc6842c2ac0249bcd36f2383b80b05122f0